src/main/java/ru/urururu/sanity/api/cfg/CfePrinter.java

```diff
@@ -210,7 +211,7 @@ private String getId(Cfe cfe) {
         if (cfe == null) {
             return "<exit>";
         }
-        return String.format("#%04d:\t", getOrCreateId(cfe, cfeIds));
+        return String.format("#%04d:\t", getOrCreateId(cfe, cfeIds)); // todo remove trailing whitespace
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)